alarm/utils.py

```diff
@@ -24,7 +23,7 @@ def any_of_trade_pair_thresholds_is_broken(trade_pair):
         return False
 
     for threshold in thresholds:
-        threshold_broken = threshold_is_broken(threshold.price, last_candle, penultimate_candle)
+        threshold_broken = threshold.is_broken(last_candle, penultimate_candle)
```

Comment:
Why do we need this check at all?

Reply:
This function check broken threshold for trade pair or not